### DISCUSSION TITLE
How do I update my version?

### DISCUSSION THREAD

**Discussion by sxdjt** [astrandb/miele#269] [Q&A] [answered]
I am running v0.1.9 of the integration but can see on the HACS page that 0.1.13 is the current release.  I've tried downloading the release .zip file and replacing the contents of custom_components/miele with its contents, but the version hasn't updated.  

I have restarted HASS and rebooted my container, but still stuck on 0.1.9.  I'm obviously not doing something right...

<img width="425" alt="Screenshot 2023-05-04 at 23 49 23" src="https://user-images.githubusercontent.com/53812540/236393605-f7dcfe79-d96f-4bca-b42e-31f39e657822.png">

**Comment by astrandb** [comment, marked as answer]
There is probably something corrupt in HACS. Try to redownload the integration in HACS.

**Comment by sxdjt** [comment]
Ah, my version of Home Assistant was out of date.  As soon as I updated, HACS prompted me to update the integration.

Running in a Docker container on a Synology, so it's a bit of a hacky install to begin with.  Thanks for the quick response!
